lib/lrama/grammar/parameterizing_rule_builder.rb

```diff
@@ -33,12 +33,10 @@ def lhs_token(token)
       end
 
       def rhs_token(token, rhs)
-        return nil unless rhs.symbol
-        term = rhs.symbol
-        @args.each_with_index do |arg, index|
-          term = token.args[index] if arg.s_value == rhs.symbol.s_value
-        end
-        term
+        return [] if rhs.symbols.empty?
+        @args.each_with_index.map do |arg, i|
```

Comment:
RHS of ParameterizingRule can contain other than parameters. For example `','` in below.

```
%rule pair(X, Y): X ',' Y { printf("(%d, %d)\n", $1, $2); }
                ;
```

Therefore this logic should be `map` over `rhs.symbols` with a block where symbol matches with an element of `@args` are replaced.
This also means line 36 can be removed.

Reply:
In other words

```
%rule pair(X, Y): X ',' Y { printf("(%d, %d)\n", $1, $2); }
                ;

%%

stmt: pair(ODD, EVEN) { printf("pair odd even\n"); }
    ;
```

Should generate `pair_ODD_EVEN: ODD ',' EVEN`. But `@args.each_with_index.map` ignores `','` because `','` is stored in `rhs.symbols`.